docs/user/openstack/deploy_baremetal_workers.md

```diff
@@ -160,34 +157,34 @@ attached to a pre-existing network. Traffic between masters and workers is route
                      openstack:
                        type: <vmComputeFlavorForWorkers>
                    replicas: 3
-         
+
                  ... other settings
-                 
+
                  platform:
                    openstack:
                      computeFlavor: <vmComputeFlavorForMasters>
 
 - Run the openshift installer:
 
         ./openshift-install create cluster --log-level debug
-        
-- wait for the installer to complete. 
+
+- wait for the installer to complete.
 
 - Once the installation is complete, enable routing between the subnet created by the installer and the preexisting subnet
-where the bare-metal machines will be attached. 
+where the bare-metal machines will be attached.
+
+- Deploy the bare-metal workers using `machinesets` and the [machine-api-operator][6]
```

Comment:
We could link to our spanking new MachineSet [upstream docs](https://github.com/openshift/installer/blob/master/docs/user/openstack/README.md#adding-a-machineset)

Reply:
Ah perfect, I was looking for that in MAO but couldn't find it.